proiect/src/business/BusinessFactory.java

```diff
@@ -0,0 +1,16 @@
+package business;
+
+import shared.Rating;
+
+public class BusinessFactory {
+	public static Business makeBusiness(String type, String name) {
```

Comment:
you can also use an enum for business type and check for the possible values here on cases

Reply:
Resolved in issue #11